File: app/cqrs/mr/readModel/detailsview/InventoryItemCreatedHandler.scala
package cqrs.mr.readModel.detailsview

import cqrs.core.Handler
import cqrs.mr.events.InventoryItemCreated
import cqrs.mr.readModel.BullShitDatabase
import cqrs.mr.readModel.InventoryItemDetailsDto

class InventoryItemCreatedHandler extends Handler[InventoryItemCreated]{

	def handle(message: InventoryItemCreated) {
		BullShitDatabase.details.put(message.id, new InventoryItemDetailsDto(message.id, message.name,0,0))
	}
}

File: app/cqrs/mr/readModel/detailsview/InventoryItemCreatedHandler.java
package cqrs.mr.readModel.detailsview;

import cqrs.core.Handler;
import cqrs.mr.events.InventoryItemCreated;
import cqrs.mr.readModel.BullShitDatabase;
import cqrs.mr.readModel.InventoryItemDetailsDto;

public class InventoryItemCreatedHandler implements Handler<InventoryItemCreated>{
	@Override
	public void handle(InventoryItemCreated message) {
		BullShitDatabase.details().put(message.id(), new InventoryItemDetailsDto(message.id(), message.name(), 0,0));
	}
}
